app/controllers/organizations_controller.rb

```diff
@@ -103,11 +103,12 @@ def setup_organization
   end
 
   def search
-    orgs_found = current_user.organizations.order(:id).where("title LIKE ?", "%#{params[:query]}%")
+    orgs_found = current_user.organizations.order(:id).where("title ILIKE ?", "%#{params[:query]}%")
+    @organization = orgs_found.page(params[:page]).per(12)
     respond_to do |format|
       format.html do
         render partial: "organizations/organization_card_layout",
-               locals: { organizations: orgs_found.page(params[:page]).per(12) }
```

Comment:
Is there a reason we're removing the pagination in the response?

Reply:
I moved it into a variable so we can grab it in the tests